extension/imos--1.0.sql

```diff
@@ -271,6 +271,28 @@ END;
 $BODY$
 LANGUAGE plpgsql; 
 
+-- Function to add ids to polygons in provided gml
+
+CREATE FUNCTION add_id_to_polygons( p_gml text )
```

Comment:
What's the naming convention here. Is it supposed to be `add_id_to_polygons` or `addIdToPolygons`?